### PR TITLE
fix(profile-member) Display message when profile member already exist

### PR DESCRIPTION
**Pull request by velcrin**
Closes [BS-15452](https://bonitasoft.atlassian.net/browse/BS-15452)

### CODE REVIEW

File: server/pom.xml [hunk 1]
```diff
@@ -311,6 +311,11 @@
                     <version>${bonita.engine.version}</version>
                     <scope>runtime</scope>
                 </dependency>
+                <dependency>
+                    <groupId>commons-httpclient</groupId>
+                    <artifactId>commons-httpclient</artifactId>
+                    <version>3.1</version>
+                </dependency>
```
Comment (abirembaut) [inline server/pom.xml]: Why the need for this dependency and isn't the version of this lib already managed in the parent pom. xml ?

**Comment by abirembaut** [comment]
See the comment above
Other than that it looks good to me

**Comment by abirembaut** [comment]
ok, I get it. It's because it is based on bonitasoft/bonita-web/pull/289

**Comment by abirembaut** [comment]
Could you rebase, please ? bonitasoft/bonita-web#289 has been merged

**Comment by abirembaut** [comment]
Not a big deal but now that I think about it, I wonder if we shouldn't use a 409 error (conflict) for this use case instead of 403. Just a though. What do you think ?

**Comment by velcrin** [comment]
Well I guess `409` would be more accurate but we use `403` in other APIs.
https://github.com/bonitasoft/bonita-web/blob/master/server/src/main/java/org/bonitasoft/web/rest/server/engineclient/UserEngineClient.java#L52

**Comment by abirembaut** [comment]
ok. I didn't remember that. Better to keep it consistent...

**Pull request merged**